src/modules/stream/cut.js

```diff
@@ -0,0 +1,272 @@
+import ffmpeg from 'ffmpeg-static';
+import { strict as assert } from 'node:assert';
+import { path as ffprobe } from 'ffprobe-static';
+
+import { spawn } from './shared.js';
+
+function mapFormat(format) {
+    if (format?.includes('webm'))
+        return 'webm';
+
+    if (format?.includes('mp4'))
+        return 'mp4';
+
+    throw new Error(`unknown format: ${format}`);
+}
+
+const close_enough = (a, b) => Math.abs(a - b) < 0.02;
+const last = arr => arr[arr.length - 1];
+const output = process => last(process.stdio);
+const lerp = (a, b, α) => Number(a) + α * (b - a);
```

Comment:
eww snake case in javascript

Reply:
![](https://github.com/wukko/cobalt/assets/136796770/b6b67477-9ee2-4381-ac80-95da7ee9fcec)